### ISSUE TITLE
Need help on reload data to table

### ISSUE THREAD

**Issue by khoinguyen1312** [tabalinas/jsgrid#756]
I have problem in reload data from API to table. In my project, table are being editing. If user click "Cancel button" table will reload data (old data) from API.
I try use $("#table").jsGrid("loadData") but it didn't work

Here is my code to create table:

```js
$("#appendix").jsGrid({
                height: "auto",
                width: "100%",
                autoload: true,
                sorting: false,
                paging: true,
                filtering: false,
                editing: true,

                controller: {
                    loadData: function () {
                        return output[1].PhuLucData;
                    }
                },
                rowClass: function (item, itemIndex) {
                    return "rowGrid";
                },
                fields: _fields,
                onItemEditing: function (args) {
                    $("#btn_Save").removeClass("disabled");
                }
            });
```
    
Thanks you.

**Comment by richraid21** [comment]
This seems like intended functionality. If the user cancels the edit, the data shouldn't be changing.

Could you rephrase your problem?

**Comment by khoinguyen1312** [comment]
@richraid21
Here is my problem, in the table, the user will have free will to edit data inner table.
 - If user click on "Save button", it will call an API, send data back to server for storing
 - If user click on "Cancel button", all data will return to the data before any editing.

Problem is, I don't want user to click on "Cancel button" on each row they are editing, I intend to let they edit all table, then decide to save all those data or not.

I also intend to save all data on the first call ajax to the server into something later I can use to load back to the table.

Thank you.

**Comment by khoinguyen1312** [comment]
Here is my fiddle, thank you.
https://jsfiddle.net/khoinguyen1312/g6xj3nod/

**Comment by tabalinas** [comment]
@khoinguyen1312, it doesn't work due to the mistake in the code (check browser Devtools).
Here is the fixed fiddle https://jsfiddle.net/6kctg215/
